census2020/census2020.ipynb

```diff
@@ -0,0 +1,500 @@
+   "metadata": {},
+   "source": [
+    "The result shows that the continental USA is overwhelmingly white in geographic terms, apart from some predominantly Hispanic regions along the Southern border, some regions with high densities of blacks in the Southeast, and a few isolated areas of Native Americans in the West.\n",
+    "\n",
+    "If you're working in a live Python environment, you can zoom into specific areas of the map, and the plot will automatically re-render to show more details at that zoom level. We can also zoom into specific regions using predefined coordinates:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "create_image(*NewYorkCity)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "create_image(*LakeMichigan)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "create_image(*Chicago)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "create_image(*Chinatown)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here we can see that the Chinatown region of Chicago has a high concentration of Asian residents as expected and Mixed residents as well, a pattern not present in the 2010 data. However, it's unclear whether the presence of the Mixed race group is due to a new classification of an already existing population, or if it reflects an entirely new demographic in the area.\n",
+    "\n",
+    "We can also observe that different cities have different racial make-ups while also being highly segregated:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "create_image(*Houston)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "create_image(*Atlanta)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "create_image(*LosAngeles)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Visualizing Racial Demographics\n",
+    "\n",
+    "We can use the `groupby` parameter to generate plots categorized by race. This approach allows us to clearly visualize the regions where each racial group predominantly resides, providing insights into the spatial patterns and demographic makeup of various areas."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "df.hvplot.points('x', 'y', rasterize=True, dynspread=True, cnorm='eq_hist',\n",
```

Comment:
This cell takes about 1 min to run for **each** selection. 

How to make it faster with hvPlot if possible?